### PR TITLE
Model Config Documents Quality Improvemrnt [res2net, shufflenet_v1, resnet, rexnet]

### PR DESCRIPTION
**Pull request by XixinYang**
params data of rexnet is still missing

### CODE REVIEW

File: configs/res2net/README.md [hunk 1]
```diff
@@ -1,19 +1,86 @@
...
+
+* Distributed Training
+
+It is easy to reproduce the reported results with the pre-defined training recipe. For distributed training on multiple Ascend 910 devices, please run
+
+```shell
+# distrubted training on multiple GPU/Ascend devices
+mpirun -n 8 python train.py --config configs/res2net/res2net_50_ascend.yaml --data_dir /path/to/imagenet
+```
+
```
Comment (geniuspatrick) [inline configs/res2net/README.md:56]: 添加--allow-run-as-root提示

File: configs/res2net/README.md [hunk 2]
```diff
@@ -1,19 +1,86 @@
...
+**Note:**  As the global batch size  (batch_size x num_devices) is an important hyper-parameter, it is recommended to keep the global batch size unchanged for reproduction or adjust the learning rate linearly to a new global batch size.
+
+* Standalone Training
+
+If you want to train or finetune the model on a smaller dataset without distributed training, please run:
+
+```shell
+# standalone training on a CPU/GPU/Ascend device
+python train.py --config configs/res2net/res2net_50_ascend.yaml --data_dir /path/to/dataset --distribute False
```
Comment (geniuspatrick) [inline configs/res2net/README.md:69]: /path/to/imagenet

File: configs/res2net/README.md [hunk 3]
```diff
@@ -1,19 +1,86 @@
...
+```
+
+### Validation
+
+To validate the accuracy of the trained model, you can use `validate.py` and parse the checkpoint path with `--ckpt_path`.
+
+```
```
Comment (geniuspatrick) [inline configs/res2net/README.md:76]: add `shell`

**Comment by XixinYang** [comment]
ShuffleNetV1_Block
resnet
res2net

**Pull request merged**